go.mod

```diff
@@ -8,15 +8,22 @@ require (
 	github.com/bmatcuk/doublestar v1.2.2 // indirect
 	github.com/fzipp/gocyclo v0.0.0-20150627053110-6acd4345c835 // indirect
 	github.com/ghodss/yaml v1.0.0
+	github.com/gobuffalo/envy v1.9.0 // indirect
```

Comment:
I'm not sure why this file is changed, you aren't adjusting the go code other than just adding 2 lines to the `TestTerraformBuiltInRules` function.